generator/src/lib.rs

```diff
@@ -85,13 +87,21 @@ named!(cfloat<&str, f32>,
     terminated!(nom::float_s, char!('f'))
 );
 
+fn khronos_link<S: Display>(name: &S) -> Literal {
+    Literal::string(&format!(
+        "<https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/{name}.html>",
+        name = name
+    ))
+}
+
 pub fn define_handle_macro() -> Tokens {
     quote! {
         #[macro_export]
         macro_rules! define_handle{
-            ($name: ident, $ty: ident) => {
+            ($name: ident, $ty: ident, $doc_link: meta) => {
```

Comment:
I think doc links should be optional. Those macros are sometimes used outside of ash and I don't think the user wants to always specify the links manually.

Reply:
Ah, good point. I've updated this to make the doc link an optional parameter.